HLTrigger/Configuration/python/HLT_75e33/modules/hltTiclTracksterLinksL1Seeded_cfi.py

```diff
@@ -0,0 +1,34 @@
+import FWCore.ParameterSet.Config as cms
+
+hltTiclTracksterLinksL1Seeded = cms.EDProducer("TracksterLinksProducer",
+    detector = cms.string('HGCAL'),
+    eid_input_name = cms.string('input'),
```

Comment:
something doesn't look right here, when comparing with the `fillDescriptions` of this producer:

https://github.com/cms-sw/cmssw/blob/eaefa4c3a28088f438f1d4b4d7a3839df859a143/RecoHGCal/TICL/plugins/TracksterLinksProducer.cc#L278-L303

I am getting runtime exceptions as:

```
----- Begin Fatal Exception 27-Sep-2024 18:45:08 CEST-----------------------
An exception of category 'Configuration' occurred while
   [0] Constructing the EventProcessor
   [1] Validating configuration of module: class=TracksterLinksProducer label='hltTiclTracksterLinksSuperclusteringDNNL1Seeded'
Exception Message:
Illegal parameters found in configuration.  The parameters are named:
 'eid_input_name'
 'eid_min_cluster_energy'
 'eid_n_clusters'
 'eid_n_layers'
 'eid_output_name_energy'
 'eid_output_name_id'
 'tfDnnLabel'
You could be trying to use parameter names that are not
allowed for this plugin or they could be misspelled.
----- End Fatal Exception -------------------------------------------------
```

Reply:
Yes, the problem is that with the PR #45821 the TracksterLinksProducer has changed and so have the related configs, which I had looked at to create this one (and the unseeded version). Will change them accordingly.